plans/completed/remove-ts-node-nestjs/remove-ts-node-from-nestjs-templates.md

```
@@ -0,0 +1,321 @@
+# Remove ts-node from NestJS Templates and Backend-v2
+
+## Problem
+
+ts-node v10.x does not support TypeScript 5.0+ array `extends` in tsconfig.json. Lisa NestJS templates use array extends (`"extends": ["./tsconfig.nestjs.json", "./tsconfig.local.json"]`), causing:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)